src/ec/packages/ec-component-site-header-harmonised/site-header-harmonised.story.js.bckp

```diff
@@ -8,6 +8,8 @@ import logo from '@ecl/ec-resources-logo/logo--en.svg';
 import siteHeaderHarmonised from './ecl-site-header-harmonised.html.twig';
 import dataGroup1 from './demo/data--group1';
 import dataGroup2 from './demo/data--group2';
+import dataGroup3 from './demo/data--group3';
```

Comment:
The file has issues with the linter, they are not appearing now because the file is renamed and not being checked, but what you should do is to rename it in story,js, run` yarn eslint --fix pathToTheFile` and then rename it again in .bckp and push it.